src/components/EditableCard.tsx

```diff
@@ -9,12 +9,15 @@ import SmartBool from '@mighty-justice/smart-bool';
 import ButtonToolbar from '../building-blocks/ButtonToolbar';
 import GuardedButton from '../building-blocks/GuardedButton';
 import { formPropsDefaults } from '../propsDefaults';
-import { ISharedFormProps } from '../props';
+import { ISharedFormModalProps, ISharedFormProps } from '../props';
```

Comment:
Not really relevant to this PR, but we can probably remove `Shared` from these titles at this point